pandas/tests/plotting/test_series.py

```diff
@@ -434,6 +434,22 @@ def test_hist_no_overlap(self):
         assert len(axes) == 2
 
     @slow
+    def test_overlapping_datetime(self):
```

Comment:
Mind moving this to `pandas/tests/plotting/test_datetimelike.py`? Seems to fit a bit better there.